oshi-core/src/main/java/oshi/hardware/HWDiskStore.java

```diff
@@ -84,6 +85,8 @@ public HWDiskStore() {
      *            Number of writes to the disk
      * @param writeBytes
      *            Number of bytes written to the disk
+     * @param queueLength
+     *            Number of I/O operations currently in progress
```

Comment:
I'd like this to be a bit more verbose. The linux doc has this longer version, something in between what you have and this would be good:

> the number of I/O requests that have been issued to the device driver but have not yet completed.

Also be clear that it's both read and write.